frontend/src/modules/admin/modules/integration/config/status/connecting.ts

```diff
@@ -0,0 +1,22 @@
+import { IntegrationStatusConfig } from '@/modules/admin/modules/integration/config/status/index';
+
+const connecting: IntegrationStatusConfig = {
+  key: 'connecting',
+  show: (integration: any) => integration.status !== undefined,
```

Comment:
Is this ok?
Currently we are showing both integrations done + in progress. Shouldn't we simply include the ones in progress?

Reply:
fixed